src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/CookieHelper.cs

```diff
@@ -31,7 +31,7 @@ public static void ProcessReceivedCookies(HttpResponseMessage response, CookieCo
                         // Ignore invalid Set-Cookie header and continue processing.
                         if (NetEventSource.IsEnabled)
                         {
-                            NetEventSource.Info(response, $"Invalid Set-Cookie '{valuesArray[i]}' ignored.");
+                            NetEventSource.Error(response, $"Invalid Set-Cookie '{valuesArray[i]}' ignored.");
```

Comment:
Same question about Info vs Error.  This is a case where the server has sent back some invalid data; is that Info or Error?

Reply:
I commented above.  But usually developers are using logging to figure out why requests fail, etc. and tend to filter for "error".  This is the same pattern we use in .NET Framework System.Net tracing where "error" is about a runtime condition where the request is failing etc.